fourcc.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+
+import sys
+import re
+
+fourcc_include = sys.argv[1]
+
+fmt_list = []
+mod_list = []
+
+fmt = re.compile(r'^#define (\w+)\s*(?:\\$\s*)?fourcc_code', flags=re.M)
+mod = re.compile(r'^#define (\w+)\s*(?:\\$\s*)?fourcc_mod_code', flags=re.M)
+
+def add_ident(l, ident_list):
+	parts = l.split()
+	ident = parts[1]
+	ident_list.append(ident)
+
+with open(fourcc_include) as f:
+	for l in f.readlines():
+		ident = fmt.search(l)
+		if ident != None:
+			add_ident(ident.group(0), fmt_list)
```

Comment:
The subgroup capture I added in that regex was so this wasn't needed. Just use `.group(1)` (or `[1]` as a shortcut).

Reply:
I used group(1) because the build bot didn't like [1]